Podfile

```diff
@@ -28,3 +28,18 @@ abstract_target 'Automattic' do
   target 'SimplenoteTests'
 
 end
+
+# Remove iOS deployment target to work around compilation issues with macOS.
+#
+# The proper fix is to update the libraries that have issues (Sentry and
+# Sodium, both dependencies of Tracks) to their latest version which addresses
+# these issues.
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+      target.build_configurations.each do |config|
+          config.build_settings['DEBUG_INFORMATION_FORMAT'] = 'dwarf'
+          config.build_settings.delete 'IPHONEOS_DEPLOYMENT_TARGET'
+          config.build_settings['ONLY_ACTIVE_ARCH'] = 'YES'
+      end
+  end
+end
```

Comment:
I cherry picked @loremattei's workaround for the macOS & arm64 issues with Xcode 12.2 from #747 . I verified the app built and was able to validate it with and upload it to App Store Connect.